modules/dxp/apps/saml/saml-api/src/main/java/com/liferay/saml/util/SamlHttpRequest.java

```diff
@@ -21,7 +21,7 @@
 /**
  * @author Michael C. Han
  */
-public interface SamlHttpRequestUtil {
+public interface SamlHttpRequest {
```

Comment:
@GabrielHenriquedev `SamlHttpRequest` looks like a class representing an HTTP request; a better name could be `SamlHttpRequestHelper`.